client/my-sites/checkout/cart/test/cart-coupon.js

```diff
@@ -176,10 +176,10 @@ describe( 'cart-coupon', () => {
 					} }
 				/>
 			);
-			applyCoupon.mockReset();
+			removeCoupon.mockReset();
```

Comment:
It's often helpful to reset/restore mocks as a beforeAll. Here's an example restoring mocks:

https://github.com/Automattic/wp-calypso/blob/3bf76e0c8e3cddf58846689fb97c7b7d6183f17a/client/state/themes/test/reducer.js#L69